continuous_integration/recipes/distributed/meta.yaml

```
@@ -27,6 +27,8 @@ requirements:
     - python >=3.8
     - pip
     - dask-core {{ dask_version }}
+    - versioneer =0.28
```

Should we make this a lower bound instead of a pin?

```suggestion
    - versioneer >=0.28
```